bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/DbLedgerStorage.java

```diff
@@ -93,7 +93,7 @@
             implements AutoCloseable {
 
         // lac
-        private Long lac = null;
+        private volatile long lac = -1;
```

Comment:
`-1L` has a meaning because `addEntry(0L)` carries -1L. I would suggest to use a different negative value, e.g. `-0xdead` or `Long.MIN_VALUE`.